code/addons/test/package.json

```diff
@@ -133,10 +133,10 @@
     "vitest": "^3.0.9"
   },
   "peerDependencies": {
-    "@vitest/browser": "^2.1.1 || ^3.0.0",
-    "@vitest/runner": "^2.1.1 || ^3.0.0",
+    "@vitest/browser": "^3.0.0",
```

Comment:
Is there a plan to change the documentation and postinstall checks in this PR or another one? Would you like me to do that?